sdk/python/examples/bayesianoptimization-katib-sdk.ipynb

```diff
@@ -0,0 +1,575 @@
+   "source": [
+    "import kubeflow.katib as kc\n",
+    "from kubeflow.katib import constants\n",
+    "from kubeflow.katib import utils\n",
+    "from kubeflow.katib import V1alpha3AlgorithmSetting\n",
+    "from kubeflow.katib import V1alpha3AlgorithmSetting\n",
+    "from kubeflow.katib import V1alpha3AlgorithmSpec\n",
+    "from kubeflow.katib import V1alpha3CollectorSpec\n",
+    "from kubeflow.katib import V1alpha3EarlyStoppingSetting\n",
+    "from kubeflow.katib import V1alpha3EarlyStoppingSpec\n",
+    "from kubeflow.katib import V1alpha3Experiment\n",
+    "from kubeflow.katib import V1alpha3ExperimentCondition\n",
+    "from kubeflow.katib import V1alpha3ExperimentList\n",
+    "from kubeflow.katib import V1alpha3ExperimentSpec\n",
+    "from kubeflow.katib import V1alpha3ExperimentStatus\n",
+    "from kubeflow.katib import V1alpha3FeasibleSpace\n",
+    "from kubeflow.katib import V1alpha3FileSystemPath\n",
+    "from kubeflow.katib import V1alpha3FilterSpec\n",
+    "from kubeflow.katib import V1alpha3GoTemplate\n",
+    "from kubeflow.katib import V1alpha3GraphConfig\n",
+    "from kubeflow.katib import V1alpha3Metric\n",
+    "from kubeflow.katib import V1alpha3MetricsCollectorSpec\n",
+    "from kubeflow.katib import V1alpha3NasConfig\n",
+    "from kubeflow.katib import V1alpha3ObjectiveSpec\n",
+    "from kubeflow.katib import V1alpha3Observation\n",
+    "from kubeflow.katib import V1alpha3Operation\n",
+    "from kubeflow.katib import V1alpha3OptimalTrial\n",
+    "from kubeflow.katib import V1alpha3ParameterAssignment\n",
+    "from kubeflow.katib import V1alpha3ParameterSpec\n",
+    "from kubeflow.katib import V1alpha3SourceSpec\n",
+    "from kubeflow.katib import V1alpha3Suggestion\n",
+    "from kubeflow.katib import V1alpha3SuggestionCondition\n",
+    "from kubeflow.katib import V1alpha3SuggestionList\n",
+    "from kubeflow.katib import V1alpha3SuggestionSpec\n",
+    "from kubeflow.katib import V1alpha3SuggestionStatus\n",
+    "from kubeflow.katib import V1alpha3TemplateSpec\n",
+    "from kubeflow.katib import V1alpha3Trial\n",
+    "from kubeflow.katib import V1alpha3TrialAssignment\n",
+    "from kubeflow.katib import V1alpha3TrialCondition\n",
+    "from kubeflow.katib import V1alpha3TrialList\n",
+    "from kubeflow.katib import V1alpha3TrialSpec\n",
+    "from kubeflow.katib import V1alpha3TrialStatus\n",
+    "from kubeflow.katib import V1alpha3TrialTemplate"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 28,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "algorithmsettings = V1alpha3AlgorithmSetting(\n",
+    "    name= \"random_state\",\n",
+    "    value = \"10\"\n",
+    "    )\n",
+    "algorithm = V1alpha3AlgorithmSpec(\n",
+    "    algorithm_name = \"bayesianoptimization\",\n",
+    "    algorithm_settings = [algorithmsettings]\n",
+    "  )\n",
+    "\n",
+    "# Metric Collector\n",
+    "collector = V1alpha3CollectorSpec(kind = \"TensorFlowEvent\")\n",
```

Comment:
For BO I think you can delete this Metrics Collector.

Reply:
deleted metrics Collector